light/proxy/routes.go

```diff
@@ -18,26 +18,27 @@ func RPCRoutes(c *lrpc.Client) map[string]*rpcserver.RPCFunc {
 		"unsubscribe_all": rpcserver.NewWSRPCFunc(c.UnsubscribeAllWS, ""),
 
 		// info API
-		"health":               rpcserver.NewRPCFunc(makeHealthFunc(c), ""),
-		"status":               rpcserver.NewRPCFunc(makeStatusFunc(c), ""),
-		"net_info":             rpcserver.NewRPCFunc(makeNetInfoFunc(c), ""),
-		"blockchain":           rpcserver.NewRPCFunc(makeBlockchainInfoFunc(c), "minHeight,maxHeight"),
-		"genesis":              rpcserver.NewRPCFunc(makeGenesisFunc(c), ""),
-		"genesis_chunked":      rpcserver.NewRPCFunc(makeGenesisChunkedFunc(c), ""),
-		"block":                rpcserver.NewRPCFunc(makeBlockFunc(c), "height"),
-		"block_by_hash":        rpcserver.NewRPCFunc(makeBlockByHashFunc(c), "hash"),
-		"block_results":        rpcserver.NewRPCFunc(makeBlockResultsFunc(c), "height"),
-		"commit":               rpcserver.NewRPCFunc(makeCommitFunc(c), "height"),
-		"data_commitment":      rpcserver.NewRPCFunc(makeDataCommitmentFunc(c), "beginBlock,endBlock"),
-		"tx":                   rpcserver.NewRPCFunc(makeTxFunc(c), "hash,prove"),
-		"tx_search":            rpcserver.NewRPCFunc(makeTxSearchFunc(c), "query,prove,page,per_page,order_by"),
-		"block_search":         rpcserver.NewRPCFunc(makeBlockSearchFunc(c), "query,page,per_page,order_by"),
-		"validators":           rpcserver.NewRPCFunc(makeValidatorsFunc(c), "height,page,per_page"),
-		"dump_consensus_state": rpcserver.NewRPCFunc(makeDumpConsensusStateFunc(c), ""),
-		"consensus_state":      rpcserver.NewRPCFunc(makeConsensusStateFunc(c), ""),
-		"consensus_params":     rpcserver.NewRPCFunc(makeConsensusParamsFunc(c), "height"),
-		"unconfirmed_txs":      rpcserver.NewRPCFunc(makeUnconfirmedTxsFunc(c), "limit"),
-		"num_unconfirmed_txs":  rpcserver.NewRPCFunc(makeNumUnconfirmedTxsFunc(c), ""),
+		"health":                    rpcserver.NewRPCFunc(makeHealthFunc(c), ""),
+		"status":                    rpcserver.NewRPCFunc(makeStatusFunc(c), ""),
+		"net_info":                  rpcserver.NewRPCFunc(makeNetInfoFunc(c), ""),
+		"blockchain":                rpcserver.NewRPCFunc(makeBlockchainInfoFunc(c), "minHeight,maxHeight"),
+		"genesis":                   rpcserver.NewRPCFunc(makeGenesisFunc(c), ""),
+		"genesis_chunked":           rpcserver.NewRPCFunc(makeGenesisChunkedFunc(c), ""),
+		"block":                     rpcserver.NewRPCFunc(makeBlockFunc(c), "height"),
+		"block_by_hash":             rpcserver.NewRPCFunc(makeBlockByHashFunc(c), "hash"),
+		"block_results":             rpcserver.NewRPCFunc(makeBlockResultsFunc(c), "height"),
+		"commit":                    rpcserver.NewRPCFunc(makeCommitFunc(c), "height"),
+		"data_commitment":           rpcserver.NewRPCFunc(makeDataCommitmentFunc(c), "beginBlock,endBlock"),
+		"data_root_inclusion_proof": rpcserver.NewRPCFunc(makeDataRootInclusionProofFunc(c), "height,beginBlock,endBlock"),
```

Comment:
Is `prove_shares` supposed to be here? I notice you have the method in `light/rpc/client.go` but nothing ever calls that?

Reply:
I guess there is a request for it:
```
func (c *baseRPCClient) DataRootInclusionProof(
	ctx context.Context,
	height uint64,
	firstBlock uint64,
	lastBlock uint64,
) (*ctypes.ResultDataRootInclusionProof, error) {
	result := new(ctypes.ResultDataRootInclusionProof)
	params := map[string]interface{}{
		"height":     height,
		"firstBlock": firstBlock,
		"lastBlock":  lastBlock,
	}

	_, err := c.caller.Call(ctx, "data_root_inclusion_proof", params, result)
	if err != nil {
		return nil, err
	}

	return result, nil
}
```

Does it need to be called somewhere else?